datadog_checks_base/datadog_checks/stubs/aggregator.py

```diff
@@ -11,11 +11,14 @@
 ServiceCheckStub = namedtuple('ServiceCheckStub', 'check_id name status tags hostname message')
 
 
-def normalize_tags(tags):
+def normalize_tags(tags, sort=False):
```

Comment:
I think you need to merge with master because those changes are already in. see https://github.com/DataDog/integrations-core/pull/1989/files

Reply:
Thanks! :)